content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,113 @@ title: Configure the Datadog Tracing Library
 
+## Common configuration options
+The following configuration options are supported across all Datadog SDKs:
+
+### Traces
+
+`DD_TRACE_<INTEGRATION>_ENABLED`
+: **Default**: `true` <br>
+**Supported Values**: Boolean <br>
+**Not Supported In**: Go SDK<br>
+**Description**: Enable or disable instrumentation for the specified `<INTEGRATION>`.
+
+`DD_TRACE_RATE_LIMIT`
+: **Default**: `100` <br>
+**Supported Values**: A positive integer
+**Description**: Sets the maximum number of traces to sample per second; applies only when either `DD_TRACE_SAMPLING_RULES` or `DD_TRACE_SAMPLE_RATE` is set.
+
+`DD_TRACE_HEADER_TAGS`
+: **Default**: `null` <br>
+**Supported Values**: A comma-separated string representing a list of case-insensitive HTTP headers, with an optional mapping to a custom tag name. Example: `User-Agent:my-user-agent,Content-Type`. <br>
+**Description**: Automatically apply specified HTTP headers as span tags. If a custom tag name is not specified, the tag key defaults to `http.request.headers.<normalized-header-name>` for request headers and `http.response.headers.<normalized-header-name>` for response headers.
+
+`DD_TRACE_ENABLED`
+: **Default**: `true` <br>
+**Supported Values**: Boolean <br>
+**Description**: Enable or disable the sending of traces to the trace agent.
+
+`DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
+: **Default**: `(?i)(?:(?:"|%22)?)(?:(?:old[-_]?|new[-_]?)?p(?:ass)?w(?:or)?d(?:1|2)?|pass(?:[-_]?phrase)?|secret|(?:api[-_]?|private[-_]?|public[-_]?|access[-_]?|secret[-_]?|app(?:lication)?[-_]?)key(?:[-_]?id)?|token|consumer[-_]?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)(?:(?:\s|%20)*(?:=|%3D)[^&]+|(?:"|%22)(?:\s|%20)*(?::|%3A)(?:\s|%20)*(?:"|%22)(?:%2[^2]|%[^2]|[^"%])+(?:"|%22))|(?:bearer(?:\s|%20)+[a-z0-9._\-]+|token(?::|%3A)[a-z0-9]{13}|gh[opsu]_[0-9a-zA-Z]{36}|ey[I-L](?:[\w=-]|%3D)+\.ey[I-L](?:[\w=-]|%3D)+(?:\.(?:[\w.+/=-]|%3D|%2F|%2B)+)?|-{5}BEGIN(?:[a-z\s]|%20)+PRIVATE(?:\s|%20)KEY-{5}[^\-]+-{5}END(?:[a-z\s]|%20)+PRIVATE(?:\s|%20)KEY(?:-{5})?(?:\n|%0A)?|(?:ssh-(?:rsa|dss)|ecdsa-[a-z0-9]+-[a-z0-9]+)(?:\s|%20|%09)+(?:[a-z0-9/.+]|%2F|%5C|%2B){100,}(?:=|%3D)*(?:(?:\s|%20|%09)+[a-z0-9._-]+)?)` <br>
+**Supported Values**: A regex string <br>
+**Description**: Applies a regex to redact sensitive data from query strings on incoming HTTP requests. Matches are replaced with `<redacted>`. If an empty string is passed, no obfuscation occurs. The resulting value is reported in the `http.url` tag. 
+
+### Diagnostics
+
+`DD_TRACE_LOG_DIRECTORY`
+: **Default**: Varies by SDK, environment, and runtime <br>
+**Supported Values**: A valid directory path that exists on the system <br>
+**Not Supported In**: Node.js, Ruby, Python SDKs<br>
+**Description**: Specifies the directory where tracer log files should be routed. If the directory does not exist, the SDK logs a warning and behaves as if the environment variable was not set.
```

Comment:
Two comments on this:
1. I don't think we should include `the SDK logs a warning and behaves as if the environment variable was not set.` This sounds a little too "technical" and not user-friendly. I think we could instead say that the SDK will fallback to its default method of diagnostic logging, or something to that effect.
2. In the Default line above, perhaps we can link to our troubleshooting docs: [/tracing/troubleshooting/tracer_debug_logs#manual-debug-log-collection](https://docs-staging.datadoghq.com/ayan.khan/doc-consistent-config/tracing/troubleshooting/tracer_debug_logs#manual-debug-log-collection)